packages/react-examples/src/react-charting/HorizontalBarChart/HorizontalBarChart.CustomCallout.Example.tsx

```diff
@@ -39,7 +39,7 @@ export const HorizontalBarChartCustomCalloutExample: React.FunctionComponent<{}>
           horizontalBarChartdata: { x: 8888, y: 15000 },
           color: DefaultPalette.redDark,
```

Comment:
Also fix the hover values in HorizontalBarChart.CustomAccessibility example

Reply:
Changes are done accordingly